components/SortAndFilterDropdowns.tsx

```diff
@@ -0,0 +1,131 @@
+import { Checkbox, FormControl, InputLabel, ListItemText, MenuItem, OutlinedInput, Select } from "@mui/material";
+import { Filter } from "../src/sortingAndFiltering";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import styles from './SortAndFilterDropdowns.module.css'
+
+// Customising the table styling using ThemeProvider
+const theme = createTheme({
+	components: {
+		MuiSelect: {
+			styleOverrides: {
+				select: {
+					fontSize: "1rem",
+					fontFamily: 'var(--primary-font-family)',
+					color: 'black',
+					'&.Mui-focused .MuiOutlinedInput-notchedOutline': {
+						borderColor: 'red',
+					},
+				},
+			}
+		},
+		MuiInputLabel: {
+			styleOverrides: {
+				root: {
+					fontSize: "1.1rem",
+					fontFamily: 'var(--primary-font-family)',
+					color: 'black',
+				}
+			}
+		}
+	}
+})
+
+
+export type SortSettings = { type: "name" | "rank" | "time" | "internal" | "external" | "total" | "users", direction: boolean }
+
+export function SortBox(sortSetting: SortSettings, handleSortChange: any) {
+
+	return <ThemeProvider theme={theme}>
+		<FormControl className={styles.sortby} sx={{ m: 1, minWidth: 220, maxWidth: 220 }}>
+		<p>Sort By</p>
+			<Select
+				value={sortSetting.type}
+				onChange={handleSortChange}
+			>
+				<MenuItem value=""> <em>None</em> </MenuItem>
+				<MenuItem value={"name"}>Name</MenuItem>
+				<MenuItem value={"rank"}>Rank</MenuItem>
+				<MenuItem value={"time"}>Time</MenuItem>
+				<MenuItem value={"internal"}>Internal count</MenuItem>
+				<MenuItem value={"external"}>External count</MenuItem>
+				<MenuItem value={"total"}>Total count</MenuItem>
+				<MenuItem value={"users"}>User count</MenuItem>
+				<MenuItem value={"mostOutdated"}>Most Outdated Dependency</MenuItem>
+			</Select>
+		</FormControl>
+	</ThemeProvider>
+}
+
+export function RankSelectionList(filterSetting: Filter, handleRankSelectionChange: any) {
+	const rankSelectionValue = [
+		...(filterSetting.showGreen ? ["green"] : []),
+		...(filterSetting.showYellow ? ["yellow"] : []),
+		...(filterSetting.showRed ? ["red"] : [])
```

Comment:
capitalize the colours because it looks off next to the other settings, use `.capitalize` 
![image](https://user-images.githubusercontent.com/59728838/189871380-d1ef7140-20a3-4e6c-81ca-037062054c35.png)

Reply:
Actually nvm, we will probably refactor this to make it user configurable